src/index.tsx

```diff
@@ -604,34 +598,28 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
   };
 
   private renderChildren = (): React.ReactNode => {
-    const { useNativeDriver, adjustToContentHeight, keyboardAvoidingBehavior } = this.props;
-    const { keyboardToggle } = this.state;
-    const marginBottom = adjustToContentHeight ? 0 : keyboardToggle ? this.handleHeight : 0;
-    const enabled = isIos && !adjustToContentHeight;
+    const { useNativeDriver, adjustToContentHeight, panGestureEnabled } = this.props;
 
     return (
       <PanGestureHandler
         ref={this.modalChildren}
+        enabled={panGestureEnabled}
         simultaneousHandlers={[this.modalContentView, this.modal]}
         shouldCancelWhenOutside={false}
         onGestureEvent={Animated.event([{ nativeEvent: { translationY: this.dragY } }], {
           useNativeDriver,
         })}
         onHandlerStateChange={this.onHandleChildren}
       >
-        <AnimatedKeyboardAvoidingView
-          behavior={keyboardAvoidingBehavior || 'position'}
-          style={{ marginBottom }}
-          enabled={enabled}
-        >
+        <Animated.View style={!adjustToContentHeight ? s.flex1 : null}>
```

Comment:
I think the issue related to the modal that goes down on Android is because of this keyboard avoiding view getting remove. But I just tried it quickly, so not sure